components/utilities/utility-icon/index.jsx

```diff
@@ -55,6 +55,8 @@ const UtilityIcon = (
 	if (path) {
 		// Use `path` prop of Icon if present
 		modifiedPath = path;
+	} else if (context.onRequestIconPath) {
+		modifiedPath = context.onRequestIconPath(category, name);
```

Comment:
Please make this an args object `{ category, name }`

Reply:
👍